gig/views.py

```diff
@@ -146,6 +146,24 @@ def form_valid(self, form):
         return result
 
 
+def SetupFormTimes(context,object):
```

Comment:
Small nit, but method names typically are underscored, while classes are initial-caps
```suggestion
def setup_form_times(context,object):
```